pkg/server/service.go

```diff
@@ -243,6 +254,22 @@ func (c *criService) register(s *grpc.Server) error {
 	return nil
 }
 
+// cniNetworkConfSyncer loads latest CNI network conf.
+func (c *criService) cniNetworkConfSyncer(du time.Duration) {
+	var tick = time.NewTicker(du)
+
+	go func() {
+		defer tick.Stop()
+
+		for {
+			if err := c.netPlugin.Load(c.cniLoadOptions()...); err != nil {
```

Comment:
suggest using a switch here and have a case for exiting the loop cleanly...

Reply:
agree we need to load once before waiting on for the duration... so when moving to a switch need that to be taken into consideration :)